app/renderers/hyrax/renderers/attribute_renderer.rb

```diff
@@ -31,7 +31,10 @@ def render
 
         attributes = microdata_object_attributes(field).merge(class: "attribute attribute-#{field}")
 
-        markup += Array(values).map do |value|
+        values_array = Array(values)
+        values_array.sort! if options[:sort]
+
+        markup += values_array.map do |value|
```

Comment:
Do we need to worry at all about modifying the order of the passed in `values`?  Would `sort` be safer?

Reply:
probably. good call. 👍🏻